extension/wasm/wasm_bindings.cpp

```diff
@@ -0,0 +1,537 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
```

Comment:
You should probably mark this API as experimental 

https://docs.pytorch.org/executorch/stable/api-life-cycle.html